### DISCUSSION TITLE
Referenziamento articoli

### DISCUSSION THREAD

**Discussion by LucaPiccolo7** [PSP-DietiEstates25/DietiEstates25#11] [Ideas]
Tab Bar vs Hamburger menu
1. https://www.lucarosati.it/blog/posizione-menu
2. https://mailchimp.com/it/resources/hamburger-menu/

**Comment by valentinapsc** [comment]
scorrimento orizzontale o verticale

1. https://bitskingdom.com/blog/vertical-vs-horizontal-layouts-rdl-web-design/

**Comment by valentinapsc** [comment]
categorie personas

1. https://www.technavio.com/report/online-auction-market-industry-size-analysis
2. https://www.cognitivemarketresearch.com/online-auction-market-report
3. http://www.fastercapital.com/content/Auction-buyer-persona--The-Art-of-Tailoring-Auctions-to-Different-Buyer-Personas.html#Bargain-hunters--collectors--investors--and-enthusiasts
4. https://us.upsellit.com/blog/the-5-types-of-online-shoppers-you-need-to-understand/
5. https://fastercapital.com/content/Car-Auction-User-Persona--Understanding-Car-Auction-User-Personas--A-Comprehensive-Guide.html
